distribution/docs/src/main/jdocs/content/_tables/conf-org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin-table-contents.adoc

```diff
@@ -0,0 +1,8 @@
+:type: documentation
+:status: published
+
+|<<org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin,Default Security Attribute Values Plugin>>
+|org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin
+|Mapping of System High-Water attributes to use in case metacards are ingested without any security marking.
```

Comment:
We should be consistent in referring to this marking set. `for system user` is above, and I don't think `System High-Water` is correct either. `System High` is more accurate, as the high-water mark is mostly used when talking about rollup of portion marks.